Семинар_7q/Task_7-1.py

```diff
@@ -0,0 +1,12 @@
+stix = list(input().split())
+glasnie = {1: "АОУЫЭЕЁИЮЯаоуыэеёиюя"}
+summ = 0
+for i in range(0, len(stix)):
+    for j in stix[i]:
+        for k, v in glasnie.items():
+            if j in v:
+                summ += k
```

Comment:
Вот тут +1 и все. Поэтому словарь не нужен.
```suggestion
                summ += 1
```